docs/APIReference-AtomicBlockUtils.md

```diff
@@ -24,3 +24,21 @@ insertAtomicBlock: function(
   character: string
 ): EditorState
 ```
+
+### moveAtomicBlockBefore
+
+```
+moveAtomicBlockBefore: function(
+  editorState: EditorState,
+  contentBlock: ContentBlock
+): EditorState
+```
+
+### moveAtomicBlockAfter
+
+```
+moveAtomicBlockAfter: function(
+  editorState: EditorState,
+  contentBlock: ContentBlock
+): EditorState
+```
```

Comment:
Yes exactly, i've tried to keep the same style for AtomicBlock functions and took insertAtomicBlock function as basis. But i absolutely agree with you, the function declarations are not very intuitive at the moment. So i've been thinking about a better way, also considered other Draft.js functions (like Modifier.moveText).

Possible solution:
-- 

```
moveAtomicBlock: function(
  editorState: EditorState,
  atomicBlock: ContentBlock,
  targetRange: SelectionState
): EditorState
```

Add a function "moveAtomicBlock". With this function you can move atomic blocks before, after or in between existing blocks.
This function does not use the actual selection as a source for the move-operation, but uses the atomicBlock, passed as a second argument, as a source for the move-operation. The atomicBlock will be moved to the targetRange (third argument). Rules:
- If targetRange.anchorOffset is 0, move the block before the block mentioned in the target range.
- If the targetRange.focusOffset is set to the end of the target block, move the atomic block after the block mentioned in the targetRange.
- If the targetRange is collapsed and targetRange.anchorOffset and targetRange.focusOffset is somewhere within the target block, split the target block and move the atomicBlock in between those.
- If the targetRange isn't collapsed replace the selection with the atomicBlock. This can also work if the targetRange includes more than one block.

Since it is not possible to determine if the atomic block should be moved before or after an empty block, we also need the functions "moveAtomicBlockBefore" and "moveAtomicBlockAfter"...

```
moveAtomicBlockBefore: function(
  editorState: EditorState,
  atomicBlock: ContentBlock,
  targetBlock: ContentBlock
): EditorState
```

Adjusted the function "moveAtomicBlockBefore". With this function you can move atomic blocks (second argument) before another block (third argument).

```
moveAtomicBlockAfter: function(
  editorState: EditorState,
  atomicBlock: ContentBlock,
  targetBlock: ContentBlock
): EditorState
```

Adjusted the function "moveAtomicBlockAfter". With this function you can move atomic blocks (second argument) after another block (third argument).